website/ui/src/app/app.module.ts

```diff
@@ -32,6 +32,8 @@ import {ProductListComponent} from './components/product-list/product-list.compo
 import {ProductPageComponent} from './components/product-page/product-page.component';
 import {ProductsPageComponent} from './components/products-page/products-page.component';
 import {TopBarComponent} from './components/top-bar/top-bar.component';
+import { LeadFormComponent } from './components/lead-form/lead-form.component';
```

Comment:
nit: remove spacing around parenthesis for consistency

Reply:
Will remove from this commit as related to a separate update